docs/architecture/adr-020-protobuf-transaction-encoding.md

```diff
@@ -105,20 +105,24 @@ message SignerInfo {
     // ModeInfo describes the signing mode of the signer and is a nested
     // structure to support nested multisig pubkey's
     ModeInfo mode_info = 2;
+    // account_sequence is the sequence of the account, which describes the
+    // number of transactions sent from a given address. It is used to prevent
+    // replay attacks.
```

Comment:
Is there really no way to change the sequence other than signing a transaction?

Reply:
What other ways do you have in mind?

For me, sequence == number of transactions in all past blocks signed by an address.

Reply:
That is true, it is incremented by 1 for every transaction that is accepted into a block with a valid signature from this account.

Some of those transactions may be errors and not changing state, but still deduct fees and update the sequence number. But all of those are in a block with a valid signature (it is gets rejected due to low fees, it doesn't make it in the block)

Reply:
I agree this is the default case, but is this guaranteed now and in the future? There can be a way to bump the sequence in order to invalidate a range of sequence numbers.

E.g. I sign a 20 transactions to be broadcasted in the future. Then I decide I don't want them to be executed. In order to do that, I send a bump sequence message to the chain which bumps my sequence by let's say 30. Then all those signatures are invalid.

Reply:
This is the way it is now and has always been. I don't see any means of change this behavior any time soon.